assertarraysubset-autoload.php

```diff
@@ -83,6 +88,27 @@ public static function load($className)
 
             return false;
         }
+
+        /**
+         * Retrieve the PHPUnit version id.
+         *
+         * As both the pre-PHPUnit 6 class, as well as the PHPUnit 6+ class contain the `id()` function,
+         * this should work independently of whether or not another library may have aliased the class.
+         *
+         * @return string Version number as a string.
+         */
+        public static function getPHPUnitVersion()
+        {
+            if (\class_exists('\PHPUnit\Runner\Version')) {
```

Comment:
Could just use `\PHPUnit\Runner\Version::class` here

Reply:
No. `:;class` is PHP 5.5 and the autoload code has to be compatible with PHP 5.4.

Reply:
I see. Missed that.

Reply:
Happens to all once of us.